iam-login-service/src/main/java/it/infn/mw/iam/core/user/DefaultIamAccountService.java

```diff
@@ -177,10 +185,21 @@ public IamAccount createAccount(IamAccount account) {
 
     eventPublisher.publishEvent(new AccountCreatedEvent(this, account,
         "Account created for user " + account.getUsername()));
-
+    
+    addToDefaultGroups(account);
     return account;
   }
 
+  private void addToDefaultGroups(IamAccount account) {
+    List<Map<String, String>> defaultGroups = iamProperties.getRegistration().getDefaultGroups();
+    if(defaultGroups != null){
```

Comment:
it seems this lines are not well formatted (no spaces between if and parenthesis e.g.)

Reply:
Done